tsconfig.json

```diff
@@ -0,0 +1,84 @@
+{
+  "include": ["./types", "./src"],
+  "compilerOptions": {
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es2015",
```

Comment:
Does this mean using `require`? I'd like to compile down to ESM, since we're using https://npm.im/esm to support native ESM loading.

Reply:
Not necessarily. This just means we use ES2015 with const, let, and all of its goodness and don't compile to an older version of JS. The module configuration is addressed by your next comment. It compiles to `esnext` which I believe is equivalent to ESM.

Reply:
@zkat, the target could be bumped higher if you want. Depends what features you want to support sans transpilation.